core/src/main/resources/lib/form/dismiss-button.jelly

```diff
@@ -0,0 +1,24 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:s="jelly:stapler" xmlns:l="/lib/layout">
+  <s:documentation> <![CDATA[
+    Jenkins themed dismiss button. This should always be
+    used instead of a custom button tag.
+    ]]>
+    <s:attribute name="name">
+      If specified, becomes the value of the name attribute.
+      When you have more than one dismiss button on the form, this can be used to determine
+      which button is pressed, as the server will get a parameter by this name.
+    </s:attribute>
+    <s:attribute name="value">
+      The tooltip text of the dismiss button.
+    </s:attribute>
+  </s:documentation>
+  <button
+    name="${attrs.name ?: 'Dismiss'}"
+    tooltip="${attrs.value ?: '%Dismiss'}"
```

Comment:
Wouldn't it make more sense to have no tooltip if not given? It would not help if it is the same as the name. (If not you should at least make it consistent to the name). Is the `%` a typo?

Reply:
% is for internationization

Reply:
Then it should be in name as well?

Reply:
name is used as part of form submission. tooltip is what's displayed to users (although as you said should probably be omitted if it's the same as `attrs.value` as it provides no value

Reply:
So anything I should change?

Reply:
Probably should remove the tooltip I think. Or make it customise-able without also changing value.

e.g.
`tooltip="${attrs.tooltip}"`

I think would work